src/sender.js

```diff
@@ -10,11 +10,12 @@ const { getNonce, getChainId } = require('./tx/web3')
 const {
   addExtraGas,
   checkHTTPS,
+  privateKeyToAddress,
   syncForEach,
   waitForFunds,
-  privateKeyToAddress
+  watchdog
 } = require('./utils/utils')
-const { EXTRA_GAS_PERCENTAGE } = require('./utils/constants')
+const { EXIT_CODES, EXTRA_GAS_PERCENTAGE } = require('./utils/constants')
 
 const { VALIDATOR_ADDRESS_PRIVATE_KEY, REDIS_LOCK_TTL } = process.env
 
```

Comment:
Should `process.exit(1)` in the line 26 use a constant from `EXIT_CODES` as well?